### ISSUE TITLE
Allow GitLab project to be created even if local directory has commits

### ISSUE THREAD

**Issue by myersjustinc** [associatedpress/datakit-gitlab#5]
* datakit-gitlab version: 0.3.0
* Python version: 3.8.2
* Operating System: macOS 10.15

### Description

A previous failed `datakit gitlab integrate` run (due to some since-resolved configuration issues) ran `git init` locally but didn't successfully create the corresponding GitLab project—and rerunning `datakit gitlab integrate` didn't do anything because it'd said the repo already was initialized.

[The code right now](https://github.com/associatedpress/datakit-gitlab/blob/77ca3dff2c50b65a77a30c3efdb23f76926f6827/datakit_gitlab/commands/integrate.py#L23-L34) assumes everything's been done correctly as long as the local project directory is a Git repository—but we probably should add some more flexibility here so existing repos can be integrated as well (or at least so errors can be handled more gracefully).

### What I Did

```
$ datakit gitlab integrate
Repo has already been initialized!
$ rm -rf .git
$ datakit gitlab integrate
Running Git initialization...
Enumerating objects: 7, done.
Counting objects: 100% (7/7), done.
Delta compression using up to 12 threads
Compressing objects: 100% (4/4), done.
Writing objects: 100% (7/7), 1.02 KiB | 1.02 MiB/s, done.
Total 7 (delta 0), reused 0 (delta 0), pack-reused 0
To gitlab.inside.ap.org:data/gitlab-test-20200722-1.git
 * [new branch]      master -> master
Project created:
	https://gitlab.inside.ap.org/data/gitlab-test-20200722-1
```

**Comment by myersjustinc** [comment]
Noting a related issue here: If the project creation process fails on GitLab for any reason, you'll hit this same problem. For example, we had someone on the team get a 502 during project creation (GitLab responded too slowly, I think), and that caused the `Repo has already been initialized!` error on subsequent attempts.

**Comment by myersjustinc** [comment]
Fixed in v0.4.0. Thanks, @larryfenn!
